tests/drivers/gpio/gpio_basic_api/boards/rpi_pico2_rp2350a_m33.conf

```diff
@@ -0,0 +1 @@
+CONFIG_SKIP_PULL_TEST=y
```

Comment:
@henrikbrixandersen 

I addressed it based on your comments received elsewhere.

https://github.com/zephyrproject-rtos/zephyr/pull/84548#discussion_r1931272839